### PR TITLE
only perform multi-root mapping if there are multiple roots

### PR DESCRIPTION
**Pull request by jonahwilliams**
## Description

In google3 a multiroot scheme is used with a single filesystem root. We need to handle the mapping slightly differently in this case

### CODE REVIEW

**Comment by jonahwilliams** [comment]
g3 fix here: 235245942

File: packages/flutter_tools/lib/src/compile.dart [hunk 1]
```diff
@@ -117,7 +117,8 @@ class PackageUriMapper {
 
     for (String packageName in packageMap.keys) {
       final String prefix = packageMap[packageName].toString();
-      if (fileSystemScheme != null && fileSystemRoots != null && prefix.contains(fileSystemScheme)) {
+      // Only perform a multi-root mapping if there are multiple roots.
+      if (fileSystemScheme != null && fileSystemRoots != null && fileSystemRoots.length > 1 && prefix.contains(fileSystemScheme)) {
```
Comment (zanderso) [inline packages/flutter_tools/lib/src/compile.dart]: Are there any line-length restrictions in the style guide?

**Review by zanderso — APPROVED** [review]
lgtm w/ question

File: packages/flutter_tools/lib/src/compile.dart (hunk 1 above)
Comment (jonahwilliams) [reply packages/flutter_tools/lib/src/compile.dart]: Generally try and keep it under 80 - in this case I agree its too far over

**Pull request merged**